### ISSUE TITLE
Backups timeout with a 504

### ISSUE THREAD

**Issue by Japhys** [linuxserver/docker-grav#22]
Been using this container for a week and very happy with it, runs very smoothly. Only one problem I cannot seem to solve: a 504 during backup.

![2022-11-24_08-26](https://user-images.githubusercontent.com/24695937/203720799-28280b5e-c36a-4491-a732-3eb22d200938.png)

My setup, all clean installs on a new vps

Linode / ubuntu 22.04
Docker / compose
Portainer
Nginx proxy manager
Docker-Grav

```
version: "2.1"
services:
  grav:
    image: lscr.io/linuxserver/grav:latest
    container_name: grav
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Europe/Amsterdam
    volumes:
      - /srv/grav/config:/config
    ports:
      - 7400:80
    restart: unless-stopped
```

I tried this is NPM
![2022-11-24_08-31](https://user-images.githubusercontent.com/24695937/203720724-b95f2a25-92cb-40b6-9149-d7d27e6e6382.png)

Any idea how to solve this? Do I need to change something in the container itself? Pretty new to Docker still so might need some help

**Comment by Japhys** [comment]
I changed so many different things but it keeps timing out at exact 60 seconds. Also used the local php.ini file and set max execution time. To no avail. So weird.

**Comment by thespad** [comment]
Do you get the same behaviour when not proxying via NPM?

**Comment by Japhys** [comment]
I tried that but logging in using my ip:port/admin gives me security token failure, I guess because it's expecting my domain. I am really new to docker and reverse proxy and in fact nginx. I just tinkered with apache and vhosts. 

I did find [this github thread]( https://github.com/NginxProxyManager/nginx-proxy-manager/issues/257)  later so I guess it does have to do with reverse proxy manager somehow. This corresponds with my problem exactly. The solution however did not work! 

It's not too big of a deal because the vps is being backed up as well and it's easy enough to back up manually
